.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 os:
   - linux
-  - osx
```

Comment:
Not sure why `osx` was removed, tests do not always pass or fail equally on both.

Reply:
nvm I forgot this was reverted.